packages/perseus/src/styles/widget-container.css

```diff
@@ -0,0 +1,113 @@
+/* Highlight states for editor/interaction */
+.perseus-widget-container.widget-nohighlight {
+    transition: all 0.15s;
+}
+
+.perseus-widget-container.widget-highlight {
+    box-shadow: 0px 0px 0px var(--wb-sizing-size_020) #ffa500;
+    transition: all 0.15s;
+}
+
+/* Start for WidgetContainer's alignment CSS styling */
```

Comment:
For the Start/End comments, can you make it easier to differentiate from the regular comments?

Something like

```
/***** Start for WidgetContainer's alignment CSS styling *****/
```

or 

```
/* ----- Start for WidgetContainer's alignment CSS styling ----- */
```